sql/utils/workflow_audit.py

```diff
@@ -237,7 +240,17 @@ def is_auto_review(self) -> bool:
         return True
 
     def generate_audit_setting(self) -> AuditSetting:
-        if self.is_auto_review():
+        auto_review = self.is_auto_review()
+        auto_review_wrong = self.sys_config.get("auto_review_wrong", "")
+        if auto_review_wrong == "":
+            auto_review_wrong = "2"
+        auto_review_wrong = int(auto_review_wrong)
+        reject = False
```

Comment:
```suggestion
        should_reject = False
```

麻烦这里名字改为 should_reject , 上方的 auto_review 不要赋值

下方的 if 条件改为 

```
if self.is_auto_review() and not should_reject:
```

这样代码的可读性会更好一点

Reply:
is_auto_review函数中会设置一个max_errlevel，表示脚本中最高级别的错误，用于判断是否可以自动pass。所以需要先运行is_auto_review，并保存返回值。